src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -397,7 +393,45 @@ function MdAutocomplete ($$mdSvgRegistry) {
             ? '<li ng-if="$mdAutocompleteCtrl.notFoundVisible()"\
                          md-autocomplete-parent-scope>' + template + '</li>'
             : '';
+      }
+
+      function getContainerType(attr) {
+        var menuContainerClass = attr.mdMenuContainerClass ? ' ' + attr.mdMenuContainerClass : '';
+
+        if (isVirtualRepeatDisabled(attr)) {
+          return '\
+            <div \
+                ng-hide="$mdAutocompleteCtrl.hidden"\
+                class="md-virtual-repeat-container md-autocomplete-suggestions-container md-whiteframe-z1' + menuContainerClass + '"\
+                ng-class="{ \'md-not-found\': $mdAutocompleteCtrl.notFoundVisible() }"\
+                role="presentation">\
+              <div class="md-autocomplete-variable-height-list-scroller" role="presentation">';
+        }
+
+        return '\
+          <md-virtual-repeat-container\
+              md-auto-shrink\
+              md-auto-shrink-min="1"\
+              ng-hide="$mdAutocompleteCtrl.hidden"\
+              class="md-autocomplete-suggestions-container md-whiteframe-z1' + menuContainerClass + '"\
```

Comment:
Should `md-virtual-repeat-container` be used here?

Reply:
I'll try it out

Reply:
It might be applied automatically, in which case it wouldn't be needed.

Reply:
To make it a bit clearer, I'm using md-virtual-repeat-container and md-standard-list-container for the two cases. What do you think?

Reply:
Sounds good.